packages/react-components/react-avatar/MIGRATION-AvatarGroup.md

```diff
@@ -6,30 +6,31 @@
 
 ## Migration from v8
 
-AvatarGroup and Facepile have similar APIs, but the main difference is how they interact with AvatarGroupItem/Persona. AvatarGroup receives AvatarGroupItems as children and places them in their respective place, this means AvatarGroup has no control over AvatarGroupItem. `size` is the only property of AvatarGroupItem that is modified by AvatarGroup to make all AvatarGroupItems the same size. Face adding functionality is not supported.
+AvatarGroup and Facepile have similar APIs, but the main difference is how they interact with AvatarGroupItem/Persona. `size` is the only property of AvatarGroupItem that is modified by AvatarGroup to make all AvatarGroupItems the same size. Face adding functionality is not supported.
```

Comment:
Since the docs exist on the public site do we still need this migration guide? Seems like duplication and we should just point people to the doc site.

Reply:
This one pointed out in the spec, but I can remove it and just point to the public docsite instead. Should we think about doing this to all components? Most have both the public docsite and package migration guides.

Reply:
I'm in favor of having a single source of truth for our documentation. Having both the docsite and migration guide increases our maitenance burden and the likelihood of the migration guide MD and docsite getting out of sync.

@GeoffCox This is something we've discussed in the past. Any thoughts?